syntax="proto3";
package auth;
option go_package = "github.com/pkusunjy/openai-server-proto/auth";

message AuthRequest {

};

message AuthResponse {
    string appid = 1;
    string secret = 2;
    string oss_endpoint = 3;
    string oss_access_key_id = 4;
    string oss_access_key_secret = 5;
};

service AuthService {
    rpc get_wx_miniprogram_token (AuthRequest) returns (AuthResponse) {}
    rpc get_oss_token (AuthRequest) returns (AuthResponse) {}
};

